inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/cls/ClsOperator.java

```diff
@@ -55,26 +56,28 @@
 @Service
 public class ClsOperator {
 
+    @Value("${cls.manager.endpoint}")
```

Comment:
Please add the configuration to `application-dev.properties` and add relevant comments.

Reply:
done